File: programmers/K번째 수.py
def solution(array, commands):
    answer = []
    
    for i in range(0,len(commands)):
        newarr = sorted([array[k-1] for k in range(commands[i][0], commands[i][1]+1)])
        answer.append(newarr[commands[i][2]-1])

    return answer

File: programmers/K번째 수.cpp
#include <iostream>
#include <stdio.h>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;
vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> arr, answer;
    for(int i=0;i<commands.size();i++){
        for(int j=array[commands[i][0]-1];j<=array[commands[i][1]-1];j++){
            arr.push_back(array[j]);
        }
        
        sort(arr.begin(), arr.end());
        answer.push_back(arr[commands[i][2]-1]);

        //arr.clear();
    }
    return answer;
}
